<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>month_line_chart_widget</class>
 <widget class="QWidget" name="month_line_chart_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_year">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <widget class="QTableWidget" name="tableWidget_month">
        <column>
         <property name="text">
          <string>月份</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>消费</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>工资</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_chart"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
